e2e/storybook-7/src/storybook-7-nested.test.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  checkFilesExist,
+  cleanupProject,
+  killPorts,
+  readJson,
+  runCLI,
+  runCommandUntil,
+  runCreateWorkspace,
+  tmpProjPath,
+  uniq,
+} from '@nrwl/e2e/utils';
+import { writeFileSync } from 'fs';
+
+describe('Storybook generators and executors for standalone workspaces - using React + Vite', () => {
```

Comment:
No need for any special settings or magic to make it work. Storybook 7 should work with pnpm, and should work with node 18.